File: Datos abstractos 28-08-25/datos_abstractos.js
class Persona {
  constructor(nombre, apellido1, apellido2) {
    this.nombre = nombre;
    this.apellido1 = apellido1;
    this.apellido2 = apellido2;
  }
}

// Uso
let p = new Persona("Pablo", "Ruelas", "López");
console.log(p.nombre, p.apellido1, p.apellido2);

File: Datos abstractos 28-08-25/datos_abstractos.py
class Persona:
    def __init__(self, nombre, apellido1, apellido2):
        self.nombre = nombre
        self.apellido1 = apellido1
        self.apellido2 = apellido2

# Uso
p = Persona("Pablo", "Ruelas", "López")
print(p.nombre, p.apellido1, p.apellido2)